bazel/grpc_objc_library.bzl

```diff
@@ -0,0 +1,114 @@
+load(
+    "//bazel:generate_objc.bzl",
+    "generate_objc",
+    "generate_objc_hdrs",
+    "generate_objc_srcs",
+    "generate_objc_non_arc_srcs"
+)
+load("//bazel:protobuf.bzl", "well_known_proto_libs")
+
+def objc_proto_grpc_library(name, deps, use_well_known_protos = False, **kwargs):
```

Comment:
Do we expect anyone to use it?

Reply:
@muxi Yeah, since there is a not-working `objc_proto_library`

Reply:
Err... I actually mean, I don't see any reference to it in the changes. Do we expect some user to use it? (I don't quite see much reason for that since if they use it with gRPC, `objc_grpc_library` generates all the proto files for them already)

Reply:
If they are not planning to use services (or no service is declared in any of the proto files), the current `objc_grpc_library` does not work because it must deterministically generate service stubs which require ARC to feed the `srcs` field... But it is doable branch on whether `srcs` is empty in `objc_grpc_library`. If it's empty then we do not pass any thing in field `srcs` for `native.objc_library` that gets invoked. So should I adapt to that?

Reply:
Yeah I should probably just do that